build_tools/benchmarks/comparisons/mobilebert_fp32_commands.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+from typing import Optional
+
+from common.benchmark_command import *
+from common.benchmark_command_factory import BenchmarkCommandFactory
+
+_DEFAULT_NUM_BENCHMARK_RUNS = 50
+_DEFAULT_NUM_THREADS = 1
+
+
+class TfliteMobilebertFP32(TFLiteBenchmarkCommand):
+  """ Specializes the benchmark command to use TFLite. """
+  def __init__(self, benchmark_binary: str, model_name: str, model_path: str,
+      test_data_dir: str, driver: str = "cpu",
+      num_threads: int = _DEFAULT_NUM_THREADS,
+      num_runs: int = _DEFAULT_NUM_BENCHMARK_RUNS,
+      taskset: Optional[str] = None):
+    super().__init__(benchmark_binary, model_name, model_path, num_threads,
+                     num_runs,
+                     taskset=taskset)
+    self.driver = driver
+    self.args.append("--input_layer=input_ids,input_mask,segment_ids")
+    self.args.append(
+        "--input_layer_value_files=input_ids:" + test_data_dir + "/input_word_id.bin,input_mask:" + test_data_dir + "/input_mask.bin,segment_ids:" + test_data_dir + "/input_type_id.bin")
+    self.args.append("--input_layer_shape=1,384:1,384:1,384")
+
+
+class IreeMobilebertFP32(IreeBenchmarkCommand):
+  """ Specializes the benchmark command to use IREE. """
+  def __init__(self, benchmark_binary: str, model_name: str, model_path: str,
+      driver: str = "dylib", num_threads: int = _DEFAULT_NUM_THREADS,
+      num_runs: int = _DEFAULT_NUM_BENCHMARK_RUNS,
+      taskset: Optional[str] = None):
+    super().__init__(benchmark_binary, model_name, model_path, num_threads,
+                     num_runs,
+                     taskset=taskset)
+    self.driver = driver
+    self.args.append("--entry_function=\"main\"")
```

Comment:
Quotes shouldn't be necessary here

```suggestion
    self.args.append("--entry_function=main")
```

Reply:
Done